### PR TITLE
wip: feat: bigint and import() has arrived stage-4 🎉

### LINKED ISSUE #348: bigint and import() arrived stage-4

### PR DESCRIPTION
**Pull request by aladdin-add**
fixes #348

### CODE REVIEW

**Comment by aladdin-add** [comment]
seems the pr in estree has not merged somehow :)
refs: https://github.com/estree/estree/pull/198

**Comment by KFlash** [comment]
I know. There also exist a 1y old open PR for BigInt that everyone follow. I think they are just a little slow :)

File: test/stage3/import/import.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ describe('Next - Import meta', () => {
     ['import(,)', Context.Empty],
     ['import(,y)', Context.Empty],
     ['import(;)', Context.Empty],
-    ['[import]', Context.Empty],
```
Comment (KFlash) [inline test/stage3/import/import.ts]: New way of fixing tests 👍

**Comment by aladdin-add** [comment]
haha, and I noticed bigint was already supported without next option. is it a bug? 🤣

**Comment by KFlash** [comment]
BigInt I guess was a bug that is now solved :)  And for the non failing test. Long time since I worked on Cherow, but I guess all you have to do is to check for parens etc after `import` in script. 

I tried it out with the Cherow REPL, and it only doesn't fail in script mode.

A lookahead maybe should do the trick?

**Comment by aladdin-add** [comment]
Thanks for this issue. This repository and project will no longer be maintained, please consider its successor: https://github.com/meriyah/meriyah 

Thanks for understanding and we apologize for any inconvenience!

**Pull request closed without merging**